projects/shared-modules/hms-video/components/hms-video/conference/conference-user-videos/conference-user-video/conference-user-video.component.ts

```diff
@@ -132,4 +147,16 @@ export class ConferenceUserVideoComponent implements OnInit, OnChanges, AfterVie
     hmsActions.leave();
     delete $event['returnValue'];
   }
+
+  toggleFollow(username: string) {
+    this.appUsersService.toggleFollow(username).subscribe(() => {
+      this.checkFollowing();
+    });
+  }
+
+  checkFollowing() {
+    if (this.currentUser && this.metaData) {
```

Comment:
Will `metaData` be defined here the first time the function gets called? Since we are using `ngOnChanges` to populate the `metaData`